synapse/replication/http/register.py

```diff
@@ -66,13 +68,14 @@ def _serialize_payload(
             "create_profile_with_displayname": create_profile_with_displayname,
             "admin": admin,
             "user_type": user_type,
+            "address": address,
         }
 
     @defer.inlineCallbacks
     def _handle_request(self, request, user_id):
         content = parse_json_object_from_request(request)
 
-        yield self.store.register(
+        yield self.registration_handler._register_with_store(
```

Comment:
Since `_register_with_store` function is being called from outside the file, can we rename it to `register_with_store` please?